switchapi/es2k/switch_pd_routing.c

```diff
@@ -180,17 +181,17 @@ switch_status_t switch_pd_nexthop_table_entry(
         goto dealloc_resources;
     }
 
-    status = tdi_key_field_set_value(key_hdl, field_id,
+    status = tdi_key_field_set_value_and_mask(key_hdl, field_id,
```

Comment:
there was a change in this API due to a defect fix. 
Please rebase your repo and resolve if any merge conflicts.

Reply:
Rebase fixed it.